future_challenges_for_RSEng.tex

```diff
@@ -431,7 +429,7 @@ \subsection{How have academic Institutions transformed by then?}
 At universities which have them, we want to see that researchers naturally accept them as part of their available services.
 Additionally researchers will seamlessly interact with infrastructures for using research software.
 \end{whatis}
-Academia, while claiming to be innovative (cite WissZeitVG here), has organisationally often adopted time-scales for change that are more reminiscient to civil services like pension funds.
+Academia, while claiming to be innovative \cite{wzvgbayern}, has organisationally often adopted time-scales for change that are more reminiscient to institutions thinking on the time scale of generations.
```

Comment:
```suggestion
Academia, while claiming to be innovative \cite{wzvgbayern}, has organisationally often adopted time-scales for change that are more reminiscent to institutions thinking on the time scale of generations.
```

(but I am not sure I understand this sentence)

Reply:
That changes in academia, often happen on time-scales of generations(human generation ~30 years)

Reply:
I decided to not bash pension funds....